src/zenml/steps/step_context.py

```diff
@@ -184,3 +184,22 @@ def get_output_artifact_uri(self, output_name: Optional[str] = None) -> str:
             Artifact URI for the given output.
         """
         return self._get_output(output_name).artifact_uri
+
+    @staticmethod
+    def _get_context_run_time() -> PipelineRunResponseModel:
+        """
+        Returns: PipelineRunResponseModel
+        """
+
+        pipeline_run_id = Environment().step_environment.pipeline_run_id
+        pipeline_run = Client().get_pipeline_run(pipeline_run_id)
```

Comment:
So far we only called `Client` in the `__init__` function which is always called in the orchestrator environment and is guaranteed to have a connection to the ZenML server. I'm not sure if this will work in all step environments, e.g., when running with a Vertex step operator. Perhaps @schustmi or @bcdurak might know this?

Reply:
Should not be an issue IMO